dojo/fixtures/test_type.json

```diff
@@ -719,6 +719,13 @@
         },
         "model": "dojo.test_type",
         "pk": 194
+    },
```

Comment:
this is useless

Reply:
should we remove this file then?

Reply:
It break some unit tests/integration tests. You could push some modifications to do that.